lib/sanitized.js

```diff
@@ -0,0 +1,14 @@
+function sanitized(array){
+    let cleanArray = [];
+    for(let i = 0; i < array.length; i++){
```

Comment:
Make sure you're using ESLint and following its suggestions. Set your default tab width to 2 spaces in VS Code's preference window.